src/bcdiceCore.rb

```diff
@@ -993,18 +998,76 @@ def rand(max)
     return value
   end
 
+  DetailedRandResult = Struct.new(:kind, :sides, :value)
+
+  # @params [Integer] max
+  # @return [Integer] 0以上max未満の整数
+  def rand(max)
+    ret = rand_inner(max)
+
+    push_to_detail(:nomal, max, ret + 1)
```

Comment:
スペルミスでしょうか？（変更する場合、テストケースも影響されます）

```suggestion
    push_to_detail(:normal, max, ret + 1)
```